compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -181,13 +181,15 @@ class Allocator {
         static_cast<uptr>(getFlags()->thread_local_quarantine_size_kb << 10));
   }
 
-  void enableRingBuffer() {
+  void enableRingBuffer() NO_THREAD_SAFETY_ANALYSIS {
+    RingBufferInitLock.unlock();
     AllocationRingBuffer *RB = getRingBuffer();
     if (RB)
       RB->Depot->enable();
   }
 
```

Comment:
One minor thing, unlock `RingBufferInitLock` at the end may be more consistent with the reversed locking order.

Reply:
Done.